tests/tasks.py

```diff
@@ -14,6 +14,7 @@
 LONG_TASK_SIGNAL_KEY = 'long_task_ok'
 
 tiger = get_tiger()
+tiger.connection.close()
```

Comment:
It still looks weird to me having the connection closed right after initializing TaskTiger.

What if `TaskTiger`'s `init()` would not use the Redis connection until it's needed? I believe the reason it's using it is for the scripts in https://github.com/closeio/tasktiger/blob/master/tasktiger/tasktiger.py#L224.

We could make `scripts` lazy:

```
class TaskTiger:   # side note: inheriting from object is no longer needed in Python 3
    def init(...):
        ...
        self._scripts = None

    @property
    def scripts(self):
        if self._scripts is None:
            self._scripts = RedisScripts(self.connection)
        return self._scripts
```

Reply:
I removed that close. I don't think it was a big cause of leaked connections and I can run tests locally without it. Would rather not make more changes in this PR not relevant to changing Python version support. I just started doing it because I couldn't run the tests locally to test 3.8 and refused to use `ulimit` to make tests pass.